core/src/window_service.rs

```diff
@@ -86,7 +91,10 @@ pub fn should_retransmit_and_persist(
         Some(bank) => leader_schedule_cache.slot_leader_at(blob.slot(), Some(&bank)),
     };
 
-    if blob.id() == *my_pubkey {
+    if !blob.verify() {
```

Comment:
_maybe_ a good idea to file an issue to possibly offload this to gpu  ?

Reply:
https://github.com/solana-labs/solana/issues/4668